MainForPR.java

```diff
@@ -0,0 +1,19 @@
+# MainForPR
+
+import java.util.List;
+
+public class Main {
+
+    public static void main(String[] args) {
+
+        List<String> names = List.of("grape", "peach", "apple", "banana", "orange");
+        List<String> sortedResult = names.stream().sorted().toList();
+        System.out.println(sortedResult);
+
+        List<String> result = names.stream()
+                .filter(name -> name.contains("p"))
+                .filter(name -> name.startsWith("a"))
+                .toList();
+        System.out.println("p���܂݁Aa����n�܂�ʕ���" + result);
```

Comment:
メソッド参照とforEachを使うとこんな感じでもかけます👍
一度試してみてください。
```suggestion
        names.stream()
                .filter(Main::containsP)
                .filter(Main::startsWithA)
                .forEach(name -> System.out.println("「p」を含み、「a」から始まる果物: " + name));
```